.custom-gcl.yml

```diff
@@ -1,6 +1,6 @@
-version:  v2.0.2
-name: golangci-kube-api-linter
+version: v2.5.0
+name: golangci-lint-kube-api-linter
 destination: ./bin
 plugins:
-- module: 'sigs.k8s.io/kube-api-linter'
-  path: ./
+  - module: 'sigs.k8s.io/kube-api-linter'
+    version: 'v0.0.0-20251029172002-9992248f8813'
```

Comment:
Probably still want this to point to the local path, this isn't an example necessarily but a way to build what's in the repo locally